### DISCUSSION TITLE
How do I remove these buttons?

### DISCUSSION THREAD

**Discussion by miltonjsvidal** [ReVanced/revanced-discussions#992] [Question] [answered]
Please, I would like to know how I remove the remix button, including that wide black space after* it, and also, that button with the letter P (it's a premium option, I know)* Thank you.

*selecting it inside the red line boxes
![fasdfsfasdfa](https://github.com/revanced/revanced-discussions/assets/72321331/c8eeae8c-d1d8-4460-a5c0-e6d020d0b078)

**Comment by KobeW50** [comment, marked as answer]
To hide the remix button, tap the profile picture > Settings > ReVanced > Layout > Hide action buttons > Hide all other action buttons, On.

You cannot remove the wide empty space in between buttons.

You currently cannot remove the Add to queue button in the feed flyout menu.

**Reply by miltonjsvidal** [reply]
Thanks man.

**Reply by valiantgenomics** [reply]
@KobeW50 Hey, any idea if/when we'll be able to remove the "Play next in queue" button?

Or maybe to make it available to revanced users?

It's highly annoying because I'm pressing it by mistake, instead of Save to Watch Later.

Would really appreciate it if there's a way to remove it entirely. 

I scoured github to find more info, and the few threads discussing this state that it's been resolved? But it doesn't seem like it.

https://github.com/ReVanced/revanced-patches/issues/2478

Please let me know if you can, thanks a lot!

**Reply by KobeW50** [reply]
> @KobeW50 Hey, any idea if/when we'll be able to remove the "Play next in queue" button?

Unfortunately, you still cannot remove the "Play next in queue" button from the feed flyout menu. Refer to this feature request regarding this

https://github.com/ReVanced/revanced-patches/issues/959



> Or maybe to make it available to revanced users?
> 
> I scoured github to find more info, and the few threads discussing this state that it's been resolved? But it doesn't seem like it.
> https://github.com/ReVanced/revanced-patches/issues/2478

That issue isn't resolved. It is just locked due to spam.
